src/dtoannotations/AdulthoodValidationProcess.java

```diff
@@ -0,0 +1,29 @@
+package dtoannotations;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+
+public class AdulthoodValidationProcess {
+
+    public String ValidateAdulthoodAnnotation (Object object) throws IllegalAccessException {
```

Comment:
Method name **MUST** start in **lowercase**
Make it **static** and you won't be creating objects of this class in **validate** method.